### PR TITLE
Solve issue: Implement a “menu” command to list out the set of possible actions that can be taken

### LINKED ISSUE #28: Implement a “menu” command to list out the set of possible actions that can be taken.
As a user, I want to be able to see the list of commands, so that I am able to know how to format my input.

### PR DESCRIPTION
**Pull request by glenn-chew**
Created MenuCommand class.

The menu lists all possible commands that can be performed, accompanied with a short description of what each command does.

Fixes #28

### CODE REVIEW

File: src/main/java/seedu/manager/command/MenuCommand.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package seedu.manager.command;
+
+/**
+ * Represents an executable menu command
+ */
+public class MenuCommand extends Command {
+    public static final String COMMAND_WORD = "menu";
+    private static final String MENU_MESSAGE = "Here are the possible commands:\n\n"
```
Comment (jemehgoh) [inline src/main/java/seedu/manager/command/MenuCommand.java:8]: Could a text block be used here instead of a string literal?

**Review by jemehgoh — COMMENTED** [review]
Overall LGTM, just one nit

File: src/main/java/seedu/manager/command/MenuCommand.java (hunk 1 above)
Comment (glenn-chew) [reply src/main/java/seedu/manager/command/MenuCommand.java]: Sure, I have updated the code.

Comment (jemehgoh) [reply src/main/java/seedu/manager/command/MenuCommand.java]: Okay, LGTM.

**Review by jemehgoh — APPROVED** [review]
Looks good to merge!

**Pull request merged**